include/ur_modern_driver/wait_for_program.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include "ros/ros.h"
+#include "ur_modern_driver/RobotModeDataMsg.h"
+// Do not forget to add ur_modern_driver to your CMakeLists.txt
+
+// Note: topic_namespace needs a leading slash.
+bool isProgramRunning(std::string topic_namespace = "")
+{
+  boost::shared_ptr<ur_modern_driver::RobotModeDataMsg const> sharedPtr;
+  ur_modern_driver::RobotModeDataMsg robot_mode_state;
```

Comment:
We're using C++11, so perhaps we should do something like (haven't tested it):

```C++
auto msg = ros::topic::waitForMessage<ur_modern_driver::RobotModeDataMsg>(..);
if (sharedPtr)
  return msg->is_program_running;
else
{
  ROS_ERROR(..);
  throw ros::Exception(..);
}
```

Reply:
Sure. It works, so I put it in like that.